common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/model/ILiferayModuleProjectProvider.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+package com.liferay.ide.core.model;
+
+import java.io.File;
+
+import org.eclipse.core.runtime.IStatus;
+
+/**
+ * @author Simon Jiang
+ */
+
+public interface ILiferayModuleProjectProvider
```

Comment:
Lets not put this interface here as there is no other interface for plugin projects.